jobserver/views/status.py

```diff
@@ -68,6 +68,11 @@ def get_stats(backend):
                 .filter(num_jobs=0)
                 .count()
             )
+            running = 0
+            pending = 0
+            for jr in backend.job_requests.all():
+                running += jr.jobs.filter(status="running").count()
+                pending += jr.jobs.filter(status="pending").count()
```

Comment:
This will execute two queries per JobRequest.  You can traverse the Job to JobRequest relationship in the filter with a `__` (double underscore) to have the database get all data you want, like this:

```
running = Job.objects.filter(job_request__backend=backend, status="running").count()
pending = Job.objects.filter(job_request__backend=backend, status="pending").count()
```

Reply:
Thank you!!

Reply:
While we're query golfing, you should be able to do it one using something like (untested):
```py
from django.db.models import Count

Job.objects.filter(
    job_request__backend=backend,
    status__in=["running", "pending"],
).values("status").annotate(count=Count("status"))
```
Which will result in a single `COUNT()` query with a `GROUP BY` on `status`.

Reply:
Very nice!  I so rarely use `.values()` that I'd not even considered you could use `.annotate()` with it 👌

Reply:
You'd struggle to call it a transparently obvious syntax, wouldn't you? I just remember needing to do this before.

Reply:
Thank you

I did experiment with this syntax when I was looking at JobRequest rather than Job. It's really useful, if hard to read. I'm not sure it's worth the additional complexity, given that performance isn't particularly important on this page?

Reply:
I'd say keep this one and add a comment to say it's getting a dictionary of status and count.  I'd certainly like an easy place to reference in the future when I need this again.  While this page isn't heavily used by external users, it is something we use for debugging so having it be performant under heavy load is always useful.

Reply:
Right, I've pushed the change now..

@ghickman Is this clear enough without needing a code comment? Is there a clearer name I can use for `counts_by_status`?